### PR TITLE
Add initial RTFD docs.

### PR DESCRIPTION
**Pull request by Decipher**
Just a copy of the README.md as a placeholder, but we need to start making a push forward with documentation and we need somewhere to start.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Overview
+
+Beetbox is a Vagrant configuration that provides your team with a versatile development environment for Drupal projects using Virtualbox for VMs (virtual machines).
+
+## Why use Beetbox?
+
+There are a number of Drupal-focussed Vagrant projects.
+
+* It is leaner than most other projects.
```
Comment (AlexSkrypnyk) [inline docs/index.md:11]: Can we please add 2 more points here (please fix wording):
- Pre-provisioned box - takes less time to kickstart the developement
- Allows granular configuration

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Beetbox - Drupal development on Vagrant [![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox)
+
+## Overview
+
+Beetbox is a Vagrant configuration that provides your team with a versatile development environment for Drupal projects using Virtualbox for VMs (virtual machines).
```
Comment (AlexSkrypnyk) [inline docs/index.md:5]: Are we sticking with 'environment for Drupal projects'? I though beetbox can be used for any LAMP/LEMP stack. I think this is a bit misleading.

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Why use Beetbox?
+
+There are a number of Drupal-focussed Vagrant projects.
+
+* It is leaner than most other projects.
+* Supports having a different VM per branch.
+* Takes advantage of Vagrant 1.8's to manage VMs for speed and disk efficiency
```
Comment (AlexSkrypnyk) [inline docs/index.md:13]: dot at the end

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+* Uses ansible for provision but it's not required on the host machine
```
Comment (AlexSkrypnyk) [inline docs/index.md:14]: dot at the end

File: docs/index.md (hunk 1 above)
Comment (Decipher) [inline docs/index.md:11]: I would say if anything that this whole file would likely be entirely re-written, as it needs to include the quickstart information... however there's no harm improving it either.

Feel free to make a PR to my fork (ensuring that if you change this file you also change README.md) or alternatively make a PR after I have accepted this one.

I would suggest that a statement like **Allows granular configuration** might need some explanation to it though.

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Beetbox - Drupal development on Vagrant [![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox)
```
Comment (AlexSkrypnyk) [inline docs/index.md:1]: Beetbox - development environment on Vagrant
(Please see my comment below).

File: docs/index.md (hunk 1 above)
Comment (AlexSkrypnyk) [inline docs/index.md:11]: I totally missed that this is a RTD and not READM.md. sight. ignore all my comments then. The all apply to both this PR and readme.md, but they can be addressed later.

**Pull request merged**